sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryClient.java

```diff
@@ -91,6 +91,9 @@ public DirectoryClient getSubDirectoryClient(String subDirectoryName) {
      *
      * {@codesnippet com.azure.storage.file.directoryClient.createDirectory}
      *
+     * <p>For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/list-directories-and-files">Azure Docs</a>.</p>
```

Comment:
>/list-directories-and-files" [](start = 73, length = 28)

Incorrect link

Reply:
Done with the checking for the entire file.